lib/commons/rscommons/national_map.py

```diff
@@ -101,9 +102,26 @@ def download_shapefile_collection(url, download_folder, unzip_folder, force_down
     return shapefiles
 
 
+def _get_metadata(item: dict):
```

Comment:
This function is just for logging? You could always use the `item.get(key_name, "Not Found")` method too... really minor point though